persistence-utils/src/com/axonivy/utils/persistence/IvyEntityManager.java

```diff
@@ -242,4 +237,11 @@ public void closeSession() {
 				closedEm ? "(the entity manager was closed)" : "");
 	}
 	
+	/**
+	 * Clears the state of this singleton instance: <b>should only be moved in JUNIT testing code!</b>
+	 */
+	public void clearThreadlocals() {
+		sessions.remove();
+		threadLocalPersistenceContexts.remove();
```

Comment:
@weissreto your proposal was correct. Cleaning these thread-locals is already enough, to make the JVM forks re-usable. Also on 10.0.0 without any change in the ivy-core.

Reply:
👍